app/models/transaction_log_entry.rb

```diff
@@ -20,7 +20,7 @@ def self.sum_balances(user)
   end
 
   def self.product_balances(user)
-    where(wallet_id: user.id).with_cents.group(:product_id).sum(:cents)
+    where(wallet_id: user.id).with_cents.group(:product_id).order('sum(cents) desc').sum(:cents)
```

Comment:
Line is too long. [96/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.